tests/env.py

```diff
@@ -0,0 +1,18 @@
+HOST = '167.114.29.32'
+PORT = 4002
+NETHASH = '578e820911f24e039733b45e4882b73e301f813a0d2c31330dafda84534ffa23'
+
+DELEGATE_NAME = 'darkjarunik'
+
+# Jarunik's devnet address for testing
```

Comment:
Why not our beloved boldninja 👀

Reply:
Because I never forget my first :laughing: